hack/boilerplate/boilerplate.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
unrelated files ?

Reply:
Boilerplate failed (so i include the fix also here and add link with the issue)